src/index.js

```diff
@@ -1,6 +1,15 @@
-const fibonacci = (n) => {
+const fibonacci = n => {
+	if (n == 1) {
+		return [1];
+	} else if (n > 1) {
+		let serie = [1];
+		for (let index = 1; index < n; index++) {
```

Comment:
el caso n = 1 está de más. Si en el for, si n = 1 no entra en el ciclo por lo que regresas únicamente [1].